.github/workflows/gh-test.yml

```diff
@@ -18,24 +18,84 @@ on:
       test-scope:
         required: true
         type: string
+      use-container:
+        required: true
+        type: boolean
+      enabled:
+        required: true
+        type: boolean
+
+env:
+  ARTIFACT_NAME: "${{ inputs.repos-name }}-${{ inputs.build-target }}-${{ inputs.sha }}"
+  ARTIFACTS_DIR: "${{ github.workspace }}/.artifacts"
 
 jobs:
-  test:
-    if: github.repository_owner == 'nv-legate'
+  test-in-container:
```

Comment:
I guess the same comment here. It seems that we are repeating pretty much the same steps, with the major difference being running in a container or not. Wouldn't we want the steps in a separate reusable workflow, and then we would just invoke the workflow from a container or without a container?

Reply:
See above comment.